coralproject/talk
# Configure enhancements

## What does this PR do?
- Fix bug when dealing with an empty Organization Contact Email https://www.pivotaltracker.com/story/show/157218143
- Fix cleanup bug causing some errors in some cases https://www.pivotaltracker.com/story/show/157218367
- Prevent reload or closing tab or window when there is pending data https://www.pivotaltracker.com/story/show/157219901

## How do I test this PR?

- Bug 1: Your organization contact email must be `null`, it should now show as an empty string instead of [object object]
- Bug 2: A series of navigating out an in the configure tab with and without pending changes should not show an error
- Feature 3: Reloading, closing tab or window with pending configure data should show a prompt
